Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Umbraco-Interface/index.md

```diff
@@ -1,28 +1,58 @@
 ---
-versionFrom: 7.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
 # Umbraco Interface
 
 ## Initial View
 
-Once you have logged in you will enter the Content section of Umbraco. There are several sections within Umbraco that allow you to manage different aspects of your site. The Content section allows you to manage the web pages of your site. Sections may vary from user to user based upon their role. As an editor or writer for instance you may only have access to the Content and Media sections. You can navigate between sections using the black section bar on the left-hand side of the screen. As you rollover one of the sections the name of the section will be revealed. Click a section to navigate to it.
+After logging in to an Umbraco V8 project you will be presented with a screen with a lot of different buttons in this guide you will gain a quick overview of what each button does.
 
-To the right of the section bar is the tree view which shows the structure of your website. To navigate the tree you can click on the black arrows to the left of tree items which will reveal any sub-pages. Clicking on the
-page name will load that page’s content into the large content area on the right of the tree view.
+![Environment.png](images/Environment.png)
 
-![section tree content.jpg](images/section tree content.jpg)
+### 1. The three circles
```

Comment:
Instead of calling it "The three circles" - how about we go with something a bit more specific to what the circles actually are? 🤔 Like, "Environments" or "Environments overview" ? What do you think?

Reply:
Hmm yeah i guess i could call it environment overview and clarify that they are represented as the three circles?

Reply:
Or perhaps "Environments Dashboard" .. 🤔 

Yes, I actually think it's already explained perfectly in the text below 💪